myhpom/tasks.py

```diff
@@ -0,0 +1,81 @@
+import json
+import requests
+from django.core.mail import send_mail
+from django.conf import settings
+from django.template.loader import get_template
+from django.template import Context
+from celery import shared_task
+from celery.task import Task
+from celery.signals import task_failure
+from myhpom.models.document import DocumentUrl
+from myhpom.models import cloudfactory
+
+
+@shared_task
+class CloudFactorySubmitDocumentRun(Task):
+    def run(self, document_url_id, document_host=None):
```

Comment:
Will we ever need to pass in document_host when its supplied by a constant of the system (staging or prod)?

Reply:
Hmmm, that's a good thought.

Reply:
hrm, oh good point. I guess...at some point when we call this we will pull the hostname from the request?

Reply:
Yes, that's what I was thinking.